src/com/lftechnology/java/training/srijan/basics/generics/GenericsInheritance.java

```diff
@@ -0,0 +1,24 @@
+package com.lftechnology.java.training.srijan.basics.generics;
+
+import java.util.logging.Logger;
+public class GenericsInheritance {
+	private static Logger LOGGER = Logger.getLogger(GenericsInheritance.class.getName());
+	public static void main(String[] args){
+		String str = "Srijan";
+		Object obj = new Object();
+		obj = str;
+		LOGGER.info("obj content: " + obj );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "obj content: " 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)